Mathlib/NumberTheory/NumberField/InfinitePlace/Embeddings.lean

```diff
@@ -0,0 +1,252 @@
+/-
+Copyright (c) 2022 Xavier Roblot. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Alex J. Best, Xavier Roblot
+-/
+import Mathlib.Algebra.Algebra.Hom.Rat
+import Mathlib.Analysis.Complex.Polynomial.Basic
+import Mathlib.NumberTheory.NumberField.Basic
+
+/-!
+# Embeddings of number fields
+
+This file defines the embeddings of a number field and, in particular, the embeddings into
+the field of complex numbers.
+
+## Main Definitions and Results
+
+* `NumberField.Embeddings.range_eval_eq_rootSet_minpoly`: let `x ∈ K` with `K` number field and
```

Comment:
```suggestion
* `NumberField.Embeddings.range_eval_eq_rootSet_minpoly`: let `x ∈ K` with `K` a number field and
```
(this might make the line be more than 100 chars, I didn't check, sorry)